cypress-shared/support/common/testcase.js

```diff
@@ -497,3 +507,40 @@ export function verifyTransactionPaymentsAPITestCase(
     }
   )
 }
+
+export function startHandlingOrder(product, env) {
+  it(`In ${product.prefix} - Start handling order`, updateRetry(3), () => {
+    cy.addDelayBetweenRetries(5000)
+    cy.getOrderItems().then((item) => {
+      cy.request({
+        method: 'POST',
+        url: startHandlingAPI(baseUrl, item[env]),
+        headers: VTEX_AUTH_HEADER(apiKey, apiToken),
+        ...FAIL_ON_STATUS_CODE,
+      }).then((response) => {
+        expect(response.status).to.match(/204|409/)
+      })
+    })
+  })
+}
+
+export function verifyOrderStatus({ product, env, status }) {
```

Comment:
Pass timeout with default value here with 10s
@shashidhar85 
When you call the functionn pass 30s

So, that this would become dynamic